src/transforms/babel.js

```diff
@@ -265,6 +265,14 @@ async function getEnvPlugins(targets, useBuiltIns = false) {
     {},
     {targets, modules: false, useBuiltIns: useBuiltIns ? 'entry' : false}
   ).plugins;
+
+  // babel-preset-env version 6.x does not cover object-rest-spread so always
```

Comment:
This does not seem like the right place for this but I'm not sure where would be a better place to put this?

Reply:
It should be fine here, but than you should probably remove it from `JSAsset.js` as now it probably runs it twice...

Reply:
On sidenote though, this would also run over node_modules, not sure if that's intended

Reply:
> It should be fine here, but than you should probably remove it from JSAsset.js as now it probably runs it twice...

The one in `JSAsset.js` seems to only be used for parser options. Inspecting the plugins array we get from `getEnvPlugins` and it does not contain `babel-plugin-transform-object-rest-spread`

> On sidenote though, this would also run over node_modules, not sure if that's intended

That is what I want since npm modules are now starting to use this feature.

Reply:
Yes, but babylon runs before the other transforms to create the ast, so this would run twice, at different places, but it'll still run twice

Reply:
That makes sense. Thanks.